.github/workflows/std.yml

```diff
@@ -24,22 +26,26 @@ jobs:
   discover:
     outputs:
       hits: ${{ steps.discovery.outputs.hits }}
-    runs-on:
-      - self-hosted
-      - discovery
+    runs-on: [self-hosted, discovery]
     steps:
       - name: Configure AWS Credentials
         uses: aws-actions/configure-aws-credentials@main
         with:
           role-to-assume: ${{ env.AWS_ROLE_ARN }}
           aws-region: ${{ env.AWS_REGION }}
+          # account is part of ecr url, thus part of `hits` output and needs to pass
           mask-aws-account-id: false
       - name: Login to Amazon ECR
         id: login-ecr
         uses: aws-actions/amazon-ecr-login@v1
+      # TODO: uncomment when nixbuild works well together with
+      #       nix daemon mode on hosted runners
+      # - uses: nixbuild/nixbuild-action@v17
+      #   with:
+      #     nixbuild_ssh_key: ${{ secrets.SSH_PRIVATE_KEY }}
+      #     generate_summary_for: job
```

Comment:
```suggestion
```

Reply:
This file is autogenerated from `../std-unexploded.yaml` so this alone wouldn't have any lasting effect.